dotcom-rendering/src/web/components/PinnedPostLiveness.importable.tsx

```diff
@@ -38,5 +62,45 @@ export const PinnedPostLiveness = ({}) => {
 			);
 		};
 	}, []);
```

Comment:
Does it matter that `pinnedPostCheckbox` is not included as a dependancy here? (in the `[]`)